modules/storytelling/story_tellers/close_to_junction_teller.h

```diff
@@ -47,6 +53,7 @@ class CloseToJunctionTeller : public BaseTeller {
   std::string yield_sign_id_;
   double yield_sign_distance_;
   StorytellingConfig config_;
+  std::shared_ptr<FrameManager> frame_manager_;
```

Comment:
is it better to save this private field in BaseTeller? otherwise we need to add this for every new type of story teller

Reply:
ok, I just not sure the frame manager is also needed by other type of teller